graphql_api/actions/components.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+from typing import Iterable, Mapping, Optional
+
+from core.models import Repository
+from graphql_api.actions.measurements import measurements_by_ids
+from timeseries.models import Interval, MeasurementName
+
+
+def component_measurements(
+    repository: Repository,
+    component_ids: Iterable[int],
+    interval: Interval,
+    after: datetime,
+    before: datetime,
+    branch: Optional[str] = None,
+) -> Mapping[int, Iterable[dict]]:
+    return measurements_by_ids(
+        repository=repository,
+        measurable_name=MeasurementName.COMPONENT_COVERAGE.value,
+        measurable_ids=component_ids,
```

Comment:
Did we need to do a similar cast to str here like we did with flags? 
`measurable_ids=[str(flag_id) for flag_id in flag_ids],`

Reply:
No `measurable_ids` takes Iterable[str] which is what the type of component_ids is. Updated the `component_ids` param type to correct value.